chocolatey/Website/Controllers/PackagesController.cs

```diff
@@ -399,7 +399,13 @@ public virtual ActionResult ListPackages(string q, string sortOrder = null, int
                 // If we are searching for something, sort by relevance.
                 sortOrder = q.IsEmpty() ? Constants.PopularitySortOrder : Constants.RelevanceSortOrder;
             }
-            
+
+            if (String.IsNullOrEmpty(moderationStatus))

```

Comment:
I usually see most code linters recommend that the `string` reserved name is used rather than the `String` type name for static method calls.

Not really sure why, but I happened to notice it. 🤷‍♂ 

```suggestion
            if (string.IsNullOrEmpty(moderationStatus))
```

Reply:
I believe it is more of a preference kind of thing than anything else.
People seem to like to use the built-in aliases instead of the concrete class/struct types, I assume because the aliases are easier to type since they are in all lowercase and sometimes shorter as well (I could be wrong here of course).

Reply:
Yeah it strikes me as a preference thing by and large. [This SO answer](https://stackoverflow.com/questions/452368/why-do-lowercase-and-uppercase-versions-of-string-exist-and-which-should-i-use) has a few more details.

Probably worth pointing out that using the alias is "safer" in some sense -- `String` as a type name could potentially register as ambiguous if you happen to (unwisely) define a type by the same name. I don't think there's a way to override the builtin `string` alias though, so that should be safe (happy to be proven wrong there though, seems there's always something weird you can do on occasion 😂).

Reply:
> I don't think there's a way to override the builtin string alias though, so that should be safe (happy to be proven wrong there though, seems there's always something weird you can do on occasion 😂).

I assume you would get a compile time error if you tried to redefine one of the aliases. Probably possible with IL manipulation (No idea how though 😄)